explore/src/clue/scala/queries/common/ObsQueriesGQL.scala

```diff
@@ -28,13 +28,15 @@ object ObsQueriesGQL:
     val document = s"""
       query($$obsId: ObservationId!) {
         observation(observationId: $$obsId) {
-          itc{
-            result {
-              exposureTime {
-                milliseconds
+          itc {
+            science {
+              selected {
+                exposureTime {
+                  milliseconds
+                }
+                exposures
+                signalToNoise
```

Comment:
Are these last 2 being used?

Reply:
We want to show S/N on the sequence table